manimlib/mobject/svg/tex_mobject.py

```diff
@@ -134,13 +134,31 @@ def organize_submobjects_left_to_right(self):
 
 
 class TexMobject(SingleStringTexMobject):
+    """
+    `manimlib.mobject.svg.tex_mobject.TexMobject`
+    TexMobject
+    ==========
+    Writes to a LaTeX file in math mode
+    (equivalent to `\\textrm{}` or `$$` in LaTeX).\n
+    Follows normal LaTeX syntax.
+    """
+
     CONFIG = {
         "arg_separator": " ",
         "substrings_to_isolate": [],
         "tex_to_color_map": {},
     }
 
     def __init__(self, *tex_strings, **kwargs):
+        """
+        CONFIG Parameters:
+        ------------------
```

Comment:
I don't know if custom sections can be used here (and actually be parsed by a documentation parser). I would guess that only the ones [here](https://numpydoc.readthedocs.io/en/latest/format.html#sections) are supported.

The CONFIG variables are a bit of a problem here since right now they can only be documented with an Attributes section but later on a Parameters section would make more sense.

Reply:
it's RST syntax

Reply:
Sounds good